src/main/java/co/inventorsoft/academy/collections/model/Range.java

```diff
@@ -1,60 +1,83 @@
 package co.inventorsoft.academy.collections.model;
 
-import java.util.Collection;
-import java.util.Iterator;
-import java.util.Set;
+import java.util.*;
```

Comment:
do not use wildcard imports. You can configure Intellij not to use wildcard imports

Reply:
Configured.